### PR TITLE
Durk's Halloween Special

### PR DESCRIPTION
**Pull request by Aidenkrz**
Progress towards #714 
Add's spooky stuff!!!

- [X] Spooky Maints Loot
- [X] Spooky Lights
- [X] Spooky Logo
- [ ] Spooky Lobby Art
- [x] Spooky Music
- [X] Spooky Parallax Backgrounds
- [X] Spooky Mob Events (ITS ALL REVENANT NOW BABY)
- [x] Spooky Paracusia
- [x] CANDY!!!!
- [x] EVIL AI (Hal 9000)
- [x] Spooky Ghost Sprite
- [X] Ghosts are more powerful during halloween so they can flicker lights more, duh.
- [x] Replace Scream with BOO
- [x] Freaky Felinids
- [X] Freaky Diona
- [X] Spooky Skeletons Roundstart
- [x] Freaky Map Vote (Re-add Train) ]
- [X] Evil Tips
- [x] Pumpkin Spice Latte
- [x] Evil Bwoink
- [x] Ghastly Death Sound
- [x] Spooky HUD Colors
- [x] Slender Mime Mob/Event
- [x] Pumpkin Slimes
- [x] FNAF Nukies
- [ ] PDA Flashlight Battery
- [ ] Spooky Meat Anom

### CODE REVIEW

File: Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml [hunk 1]
```diff
@@ -102,6 +102,12 @@
     - FootstepSound
     - DoorBumpOpener
     - VulpEmotes
+  - type: Paracusia
```
Comment (Piras314) [inline Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:105]: add this to BaseMobSpeciesOrganic, not every single one

Comment (Aidenkrz) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:105]: Not all of them use BaseMobSpeciesOraganic I'm pretty sure

Comment (Aidenkrz) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:105]: yeah felinids dont use it

Comment (Piras314) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:105]: @Aidenkrz felinids do, they inherit from human which inherics from it

Comment (Aidenkrz) [reply Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml:105]: oh :godo:

**Comment by Verbalase** [comment]
wait, this might be fire?

**Comment by Aidenkrz** [comment]
image
image
:trollface:

**Comment by Aidenkrz** [comment]
Also i need more ideas for stuff to add

**Comment by Aidenkrz** [comment]
@coderabbitai review

**Comment by BombasterDS** [comment]
Replace all slimes with live pumpkins (when you someone sprite it :godo:)

**Comment by InfinityPandaRed** [comment]
i can try to do some sprites for this

> Replace all slimes with live pumpkins (when you/someone sprite it :godo:)

**Comment by Aidenkrz** [comment]
> i can try to do some sprites for this
> 
> > Replace all slimes with live pumpkins (when you/someone sprite it :godo:)

someone beat you to it

**Comment by InfinityPandaRed** [comment]
> 
> someone beat you to it

yeah that was me

**Comment by BombasterDS** [comment]
What you thinking about replacing syndicate war declaring sound to fnaf jumpscare sound? (FNAF 6 jumpscare sound fit's better than others imo) https://www.youtube.com/watch?v=ix-dXbyvSpg

**Comment by CatMagic1** [comment]
This is actually really cool

**Comment by Piras314** [comment]
its spooky time, im drafting this pr into the halloween branch

**Pull request merged**